src/components/button/index.njk

```diff
@@ -55,14 +55,28 @@ Groups are used to group two or more buttons together and easily align them alon
 }}
 
 ### Ghost
-The ghosted button is differently styled button and has been designed to be used in a header. This then could be used to allow the user to save and logout of the current context.
+The ghosted button is differently styled button and has been designed to be used in a header or footer. This then could be used to allow the user to save and logout of the current context.
```

Comment:
```suggestion
The ghosted button style is for use in the [header](/components/header/#external-with-language-and-save-button) or footer. This can be used to allow a user to save and sign out of the current context.
```